### DISCUSSION TITLE
Social Media Image Not Working

### DISCUSSION THREAD

**Discussion by stevebarakat** [johnlindquist/kit#445] [Errors] [answered]
I posted a link to Scriptkit on Discord today and the default image did not make the app look appealing, so I removed the preview.
![kit](https://user-images.githubusercontent.com/3830326/134416864-87995111-488f-442e-b97a-625f0157a781.png)

**Comment by johnlindquist** [comment, marked as answer]
Thanks! Got this sorted. Discord was a showing 415 (unsupported media) status code while everywhere else it works just fine. Moved the same image to cloudinary and it's working...

![CleanShot 2021-09-23 at 14 53 34](https://user-images.githubusercontent.com/36073/134582399-20d03c62-0e4a-420d-91f8-f421a2c678ce.png)
